src/components/renderer/form-loop.vue

```diff
@@ -1,18 +1,16 @@
 <template>
   <div class="form-group">
-    <div v-for="loopIndex in times" :key="loopIndex">
-      <form>
```

Comment:
This `form` element was the cause of the issue. Because nested buttons didn't specify a `type`, they defaulted to `"submit"`, causing this form to submit and reload the page.

I also moved the `v-for` from the `div` to `vue-form-renderer` and removed that redundant `div`.